### PR TITLE
Cleanup goreportcard warnings

### PR DESCRIPTION
**Pull request by gliptak**
https://goreportcard.com/report/github.com/constabulary/gb

### CODE REVIEW

File: cgo.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func cgo15(pkg *Package) (*Action, []string, []string, error) {
 	cgoLDFLAGS = append(cgoLDFLAGS, pcLDFLAGS...)
 
 	runcgo1 := []*Action{
-		&Action{
```
Comment (davecheney) [inline cgo.go]: No pe

Comment (davecheney) [inline cgo.go]: Sorry, nope

Comment (davecheney) [inline cgo.go]: Make this look like executor_test below

File: internal/debug/debug.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// debug provides a light weight debug facility.
+// Package debug provides a light weight debug facility.
```
Comment (davecheney) [inline internal/debug/debug.go]: Nope

Comment (davecheney) [inline internal/debug/debug.go]: Nope, this is redundant

**Comment by gliptak** [comment]
Sure, removed changes pointed out.

**Comment by davecheney** [comment]
Thanks.

**Pull request merged**